server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java

```diff
@@ -162,12 +163,13 @@ protected boolean doStart(JobContext context) throws Exception {
             }
         }
         result.setTotalStatements(index);
-        log.info("The sql plan task execute finished,result={}", result);
 
         // all sql execute csv file list write to zip file
         writeZipFile();
         // upload file to OSS, also contains error record where is non-null
         upload();
+
+        log.info("The sql plan task execute finished,result={}", result);
```

Comment:
only print the SQL that failed to execute

Reply:
Have been changed.